docs/tech/compatibility-management.md

```diff
@@ -0,0 +1,77 @@
+# BDK compatibility management
+
+You can check our existing change management policy for our HTTP API [here](https://developers.symphony.com/restapi/docs/api-change-management).
+
+## BDK services
+
+BDK services is a layer in BDK which are the wrappers of all the REST api endpoints provided by SBE and Agent. 
+These services provide bot developers a more friendly way to interact with SBE and Agent. 
+Developers now no more need to precisely create a http call to the endpoint which is now under 
+mission of the BDK services layer.
+
+The BDK services relies on the many Apis classes which are generated automatically by Swagger 
+OpenAPI from several swagger specs files describing all the endpoints provided by SBE/Agent. 
+[Symphony-api-spec](https://github.com/symphonyoss/symphony-api-spec)
+
+## Symphony-api-spec
+
+Symphony-api-spec is a project that contains the specifications of all the endpoints exposed 
+by SBE and Agent. 
+These specifications are divided into 4 major parts: agent, pod, authenticator, login.
+
+Each part contains 2 files: 
+
+* xx-api-public.yaml : contains the information of the endpoints which are currently active on SBE/Agent.
+
+* xx-api-public-deprecated.yaml : a superset of the xx-api-public.yaml which contain the information of all the endpoints which are both active and deprecated on SBE/Agent.
+
+The Api classes in BDK will be generated from these specifications and then be used in BDK services and 
+BDK authenticator (a special BDK service).
+
+## BDK compatibility based on the version of Symphony-api-spec
+Because the BDK used the Symphony-api-spec for generating the Api classes, then the compatibility between 
+BDK and SBE/Agent will depend on the version of Symphony-api-spec that the BDK is using and the compatibility between 
+the specs and the SBE/Agent.
+
+From the official version 2.0.0 of the BDK, we start using the version branch of the Symphony-api-spec 
+(20.9 for the BDK 2.0.0), it means that the version of the BDK will compatible with the SBE product version 20.9.0.
+
+## API change management
+
+Each BDK version will be tied to one specific product version corresponding to one version of api specifications, 
+and the version of supported SBE and Agent can be found in [Agent compatibilities](https://developers.symphony.com/restapi/docs/agent-compatibilities).
+
+| Version         | Supported by BDK |
```

Comment:
maybe use Product Version to align with the agent compat table (I would drop SBE because it is too specific)